samples/contrib/azure-samples/kfp-azure-databricks/databricks/_job_op.py

```diff
@@ -0,0 +1,312 @@
+import json
+from kfp.dsl import ResourceOp
+
+class CreateJobOp(ResourceOp):
+    """Represents an Op which will be translated into a Databricks Job creation
+    resource template.
+
+    Example:
+
+        import kfp.dsl.databricks as databricks
+
+        databricks.CreateJobOp(
+            name="createjob",
+            job_name="test-job",
+            spec={
+                "new_cluster" : {
+                    "spark_version":"5.3.x-scala2.11",
+                    "node_type_id": "Standard_D3_v2",
+                    "num_workers": 2
+                },
+                "libraries" : [
+                    {
+                        "jar": 'dbfs:/my-jar.jar'
+                    },
+                    {
+                        "maven": {
+                            "coordinates": 'org.jsoup:jsoup:1.7.2'
+                        }
+                    }
+                ],
+                "timeout_seconds" : 3600,
+                "max_retries": 1,
+                "schedule":{
+                    "quartz_cron_expression": "0 15 22 ? * *",
+                    "timezone_id": "America/Los_Angeles",
+                },
+                "spark_jar_task": {
+                    "main_class_name": "com.databricks.ComputeModels",
+                },
+            }
+        )
+
+        databricks.CreateJobOp(
+            name="createjob",
+            job_name="test-job",
+            new_cluster={
+                "spark_version":"5.3.x-scala2.11",
+                "node_type_id": "Standard_D3_v2",
+                "num_workers": 2
+            },
+            libraries=[
+                {
+                    "jar": 'dbfs:/my-jar.jar'
+                },
+                {
+                    "maven": {
+                        "coordinates": 'org.jsoup:jsoup:1.7.2'
+                    }
+                }
+            ],
+            timeout_seconds=3600,
+            max_retries=1,
+            schedule={
+                "quartz_cron_expression": "0 15 22 ? * *",
+                "timezone_id": "America/Los_Angeles",
+            },
+            spark_jar_task={
+                "main_class_name": "com.databricks.ComputeModels",
+            }
+        )
+    """
+
+    def __init__(self,
+                 name: str = None,
+                 k8s_name: str = None,
+                 job_name: str = None,
+                 spec: {}=None,
```

Comment:
Better to add space around '=' to make code style consistent.

Reply:
Sure, that is what I did at first. But then linter complains with message: "No space allowed around keyword argument _pylint(bad-whitespace)_", but only for {}=None and not for e.g. str=None.

Reply:
Made the change and added the spaces.